conf/york_viking_slurm.config

```diff
@@ -0,0 +1,51 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Nextflow config file for York Viking Cluster for the SLURM login nodes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Author: Matthew Care
+Mail: matthew.care@york.ac.uk
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+params {
+    config_profile_contact = "Matthew Care"
+    config_profile_description = "The University of York Viking profile -- SLURM login nodes"
+    config_profile_url = "https://vikingdocs.york.ac.uk/"
+}
+
+apptainer {
+    enabled = true
+    autoMounts = true
+    // the default is 20 minutes and fails with large images
+    pullTimeout = "3 hours"
+}
+
+process{
+    maxRetries = 3
+    clusterOptions = "--get-user-env --account=${System.getenv('USER_ACCOUNT')}" // Get user environment and assign account
+    errorStrategy = { ( task.exitStatus == 0 ) ? "retry": "terminate"
+    }
```

Comment:
```suggestion
process {
    maxRetries = 3
    clusterOptions = "--get-user-env --account=${System.getenv('USER_ACCOUNT')}" // Get user environment and assign account
    errorStrategy = { ( task.exitStatus == 0 ) ? "retry": "terminate"
}
```

Reply:
Not sure I follow the errorStrategy, exitStatus 0 isn't a failure, so why would you want to retry on that?

Reply:
You are correct that this is confounding.  This config is based on https://github.com/nf-core/configs/blob/master/conf/ebi_codon_slurm.config which seemed a good working basis for a slurm config.  However,  you're correct that it would make more sense with a task.exitStatus != 0.

Reply:
OK, I think for now it's better to leave the pipeline defaults to handle failures like this (as it covers most common exit codes). If you get lots of user reports could add it back in again.